lib/mutant/mutator/node/block.rb

```diff
@@ -36,6 +36,38 @@ def mutate_body
           return unless body
           emit(body)
           emit_body_mutations
+
+          mutate_body_receiver
+        end
+
+        # Mutate method send in body scope of `send`
+        #
+        # @return [undefined]
+        #
+        # @api private
+        def mutate_body_receiver
+          return unless body.type.equal?(:send)
+
+          emit(s(:send, send_receiver, body_selector))
```

Comment:
This does not forward the arguments. Would turn:

```
foo { bar(baz) }
```

Into:

```
bar
```

Where I think we want:

```
bar(baz)
```